README.md

```diff
@@ -71,10 +71,14 @@ disconnects or multi-threaded usage. This can be done using the `r2d2` feature.
 redis = { version = "0.27.5", features = ["r2d2"] }
 ```
 
-For async connections, connection pooling isn't necessary. The `MultiplexedConnection` is
-cloneable and can be used safely from multiple threads, so a single connection can be easily
-reused. For automatic reconnections consider using `ConnectionManager` with the `connection-manager` feature.
+For async connections, connection pooling isn't necessary, unless blocking commands are used.
+The `MultiplexedConnection` is cloneable and can be used safely from multiple threads, so a 
+single connection can be easily reused. For automatic reconnections consider using 
+`ConnectionManager` with the `connection-manager` feature.
 Async cluster connections also don't require pooling and are thread-safe and reusable.
+If blocking commands are used, the multiplexing won't help since the server won't handle commands
```

Comment:
```suggestion

Multiplexing won't help if blocking commands are used since the server won't handle commands
from blocked connections. If you want to be able to handle commands concurrently with blocking commands,
you should send them on another connection. 
```
... or something like that. One way or another, I would break the last sentence into two.

Reply:
I'm fine with this phrasing.